### PR TITLE
Companion for substrate #13121 - BEEFY Equivocations support

### PR DESCRIPTION
**Pull request by acatangiu**
Implement new equivocations `BeefyApi` and new `pallet_beefy::Config` for Rococo. All other runtimes get dummy implementations while BEEFY is not deployed.

Companion for: https://github.com/paritytech/substrate/pull/13121

### CODE REVIEW

File: runtime/rococo/src/lib.rs [hunk 1]
```diff
@@ -1438,7 +1452,7 @@ construct_runtime! {
 		// Rococo specific pallets (not included in Kusama). Start indices at 240
 		//
 		// BEEFY Bridges support.
-		Beefy: pallet_beefy::{Pallet, Storage, Config<T>} = 240,
+		Beefy: pallet_beefy::{Pallet, Call, Storage, Config<T>} = 240,
```
Comment (ordian) [inline runtime/rococo/src/lib.rs:1455]: Doesn't it need `ValidateUnsigned`?

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2292299

File: runtime/rococo/src/lib.rs (hunk 1 above)
Comment (acatangiu) [reply runtime/rococo/src/lib.rs:1455]: right, good catch! fixed.

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2293818

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2310415

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2355751

**Review by ordian — APPROVED** [review]

**Review by sandreim — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2383331

**Comment by acatangiu** [comment]
@paritytech/locks-review ptal at this small companion

**Comment by acatangiu** [comment]
bot rebase

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2403915

**Review by andresilva — APPROVED** [review]

**Pull request merged**